packages/homepage/content/users.json

```diff
@@ -102,5 +102,10 @@
+  },
 ]
```

Comment:
It's a json file you can't add an ending comma :(

Reply:
Fixed that for you!